package.json

```diff
@@ -36,6 +36,10 @@
     {
       "name": "Serkan Inci",
       "email": "serkani@microsoft.com"
+    },
+    {
```

Comment:
This seems odd to add yourself while bumping a development version

Reply:
Is it? No one else seems to have taken the initiative. Also, I plan on doing more work later with some iteration path ideas.